app/src/main/java/org/greenstand/android/TreeTracker/activities/MainActivity.kt

```diff
@@ -73,10 +73,10 @@ class MainActivity : AppCompatActivity(), ActivityCompat.OnRequestPermissionsRes
         if (mSharedPreferences!!.getBoolean(ValueHelper.FIRST_RUN, true)) {
```

Comment:
if mSharedPreferences could be null here it is better to do a null check instead of !!

Reply:
On 73 line (MainActivity) did you said that I should replace the line `if (mSharedPreferences!!.getBoolean(ValueHelper.FIRST_RUN, true)) {` with `if (mSharedPreferences**?.**getBoolean(ValueHelper.FIRST_RUN, true)) {` Because When I do this inside the if statement is a Type mismatch, we will have a Boolean? instead of a Boolean type that is required.